### ISSUE TITLE
Create 'Community' collective

### ISSUE THREAD

**Issue by gusaus** [communitycontributorgroup/opencollective#6]

**Comment by ShaneCurcuru** [comment]
Just to clarify: this task is to create an actual "collective" at OC, correct?
https://opencollective.com/create

What are the actual questions/answers needed to create a collective, and how would our future community group decide who's on the list, who's in charge, or any other details?  (I haven't had time to create a username at OC yet, sorry)

**Comment by gusaus** [comment]
Yes! I was planning on filling in a bit more detail in the description. But to answer your question - I 'think' this would be a case where (similar to https://github.com/communitycontributorgroup/opencollective/issues/3) someone from Open Collective would create it for us (assuming they were supportive of the idea). 

With regards to how we'd manage, that's something we'd detail and document in https://github.com/communitycontributorgroup/opencollective/issues/1

**Comment by gusaus** [comment]
Instead of rambling on about what's been happening in the background since the last update, I'll reference [this conversation](https://opencollective.com/community/conversations/open-collective-community-lpye3g6a) (one way, so far) I started on the [Open Collective community collective](https://opencollective.com/community/).

A quick skim through the conversation and the [About](https://opencollective.com/community#category-ABOUT) should provide background and context. 

For those unaware, @xdamman is a Open Collective co-founder and the visionary behind [Open Collective cities](https://blog.opencollective.com/turning-our-city-into-an-open-collective/), the catalyst for [what we're going to enable](https://github.com/orgs/communitycontributorgroup/discussions/2)!

**Comment by gusaus** [comment]
I've been corresponding with @xdamman (owner/admin of https://opencollective.com/community) about teaming up to   

- co-manage/admin the collective
- extend by creating [projects](https://docs.opencollective.com/help/collectives/projects) for community-governed working groups, projects, interest areas, and locations (regions/countries/cities) we're outlining in [Open Collective Community google doc](https://docs.google.com/document/d/1nZR31Xfa9eWz1-2KQ9PGNfxjJdJLm45YoCSnOw13K8I/edit?usp=sharing)
- revisit/reimagine the _community_ vision on https://opencollective.com/community#category-ABOUT ([see this conversation](https://opencollective.com/community/conversations/open-collective-community-lpye3g6a))

Considering the collective would essentially be representing Open Collective's Community - we'll need Open Collective leadership involved. Especially if we're enabling a community they can Exit to https://opencollective.com/e2c

**Comment by tekNorah** [comment]
@gusaus have we yet reached OC leadership to discuss this? Any updates here?

**Comment by gusaus** [comment]
@tekNorah I believe I reached out to @xdamman soonafter the [OCF dissolution](https://blog.opencollective.com/open-collective-official-statement-ocf-dissolution/) was announced. Haven't heard anything back since we corresponded last year.

I think the loss of their nonprofit Slack instance (I'm assuming that's directly related to the OCF dissolution) creates an additional need for an engaged (sustainable) community. 

Access to https://opencollective.com/community (or similarly named collective should Xavier decide to keep https://opencollective.com/community as is) will provide us with the mechanism to do that.

**Comment by gusaus** [comment]
> the current OC Slack instance isn't going away, just the number of members will be limited and a separate OC 'Community' workspace will be created, which I think fills the need well

I was repeating the wordage in the [Slack thread](https://opencollective.slack.com/archives/C0HSLRNVC/p1715867605552449). Looks like the OC team is still evaluating options, but the point I was trying to make is that providing input, to helping setup and manage (https://github.com/fosscontributors/opencollective/issues/2 is related) are additional ways members of the Community Working Group can contribute to Open Collective's community (https://github.com/orgs/fosscontributors/projects/2). 

> How does facilitation of the [OC | Community](https://opencollective.com/community) collective help to solve this problem?

It will provide the mechanism to raise funds and pay contributors (https://github.com/fosscontributors/opencollective/issues/4 is related to the latter).
